src/main/resources/application.properties

```diff
@@ -14,16 +14,16 @@ hibernate.id.new_generator_mappings=true
 spring.mvc.pathmatch.matching-strategy: ANT_PATH_MATCHER
 
 server.port=8081
-logging.file.name=//users//nbashyam//Downloads//ecrNow.log
+logging.file.name=logs/ecrNow.log
 
 # Used for systemLaunch API for app implementors of Release 2.0 only
-ersd.file.location=//users//nbashyam//Downloads/ersd.json
+ersd.file.location=eRSDv2.json
 
-schematron.file.location=//users//nbashyam//Downloads//CDAR2_IG_PHCASERPT_R2_STU1.1_SCHEMATRON.sch
-xsd.schemas.location=//users//nbashyam//Downloads//schemas//CDA_SDTC.xsd
+schematron.file.location=${SCHEMATRON_LOCATION:CDAR2_IG_PHCASERPT_R2_STU1.1_SCHEMATRON.sch}
+xsd.schemas.location=${XSD_SCHEMA:CDA_SDTC.xsd}
```

Comment:
Since both of these are locations, shouldn't this one also have "LOCATION" in it's name? Maybe `SCHEMA_LOCATION`?

Reply:
Sounds good